### ISSUE TITLE
[Catalog] Implement A11yCatalog

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#4353]
Create an Accessibility app target similar to MDCCatalog and MDCDragons.

We would like to test accessibility of our components
The easiest way to do this may be with custom interfaces / view controllers designed with the assistance of our A11y team.
These should *not* be in our standard catalog.
For ease of scripting, testing and demonstration these interfaces will need their own app target.
The view controllers may be visible in both Dragons and Accessibility.

---
#### Internal data
- Associated internal bug: [b/117178108](http://b/117178108)

**Comment by ianegordon** [comment]
Create an Accessibility app target similar to MDCCatalog and MDCDragons.

We would like to test accessibility of our components
The easiest way to do this may be with custom interfaces / view controllers designed with the assistance of our A11y team.
These should *not* be in our standard catalog.
For ease of scripting, testing and demonstration these should *not* be in Dragons.

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
